### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by Ibizugbe**
In this pull request, I have completed the first milestone of the project, adhering strictly to the specified requirements.
- The project is professionally documented.
- There are no linter errors
- JavaScript best practice has been used.
- React app has been initialized using create react app.
- The app contains a folder that has reusable components.
- The app uses react-router.
- The app has two routes, linking to the books section and the category section.
- The app has a form component for adding books.
- The app has a book component for displaying a single book and receives a title and author as props.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+- React (CRA)
+
+## Milestones
+
+- [x] Milestone 1: **Initialize project with components (React only).**
+- [x] Milestone 2: **Add reducers and actions.**
+- [x] Milestone 3: **Use Redux in React components.**
+- [x] Milestone 4: **Connect to API.**
+- [x] Milestone 5: **Styling**
+
```
Comment (zieeco) [inline README.md]: - Please, kindly remove this section as they're not relevant to the project and is not a section in the readme file.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+#### Use the live Website
+
+[Bookstore Website (Netlify)]()
+
+### **Option 2**
+
+#### Set up your own copy locally
+
+- Clone [the GitHub Repository](https://github.com/Ibizugbe/bookstore)
+- Go to the Project `cd math-magicians`
```
Comment (zieeco) [inline README.md]: - Please, kindly update the name of your project here.

File: src/components/Nav.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        </ul>
+        <ul>
+          <li>
+            <Link to="/" className="nav-links">
+              Books
+            </Link>
+          </li>
+          <li>
+            <Link to="/components/Catigory" className="nav-links">
+              Catigory
```
Comment (zieeco) [inline src/components/Nav.js]: - Please, kindly correct your spelling for the category.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @<STUDENT NAME>,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Project set up correctly. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️

- [ ] Please kindy correctly the category spelling of your component.

_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Nav.js (hunk 3 above)
Comment (Ibizugbe) [reply src/components/Nav.js]: Thanks for the observations, I will resolve them quickly.

**Review by Ismailco — APPROVED** [review]
### Hi @Ibizugbe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibizugbe** [comment]
Thanks for the review

**Pull request merged**